final version of README

nab1o opened the pull request.

Commit:
final version of README

platisd commented on README.md:
```diff
@@ -1 +1,84 @@
\ No newline at end of file
+[1. Project Overview](https://github.com/DIT112-V20/group-01#project-overview)
+* [what it the idea of our project?](https://github.com/DIT112-V20/group-01#what-is-the-idea-of-our-project)
+* [Why will we make it?](https://github.com/DIT112-V20/group-01#why-will-we-make-it)
```
You don't have to include the entire URL Just the part with the `#why-will-we-make-it`should be enough.

Commit:
added app images

Commit:
Final version i promise

Included screenshots of the app in the user manual, and added wifi library to installation part

ghost commented:
Resolves #56
